### PR TITLE
feat: Added complaint button, hidden as an easter egg 'complaint'.

### PR DESCRIPTION
**Pull request by LiadOvdat5**
New Feature
# Description
When opened you can write your info and submit.
to show the button, write 'complaint'

## screenshots
image
image

### CODE REVIEW

**Review by NoamGaash — COMMENTED** [review]
good job! do you need any help with the linting warnings?

**Comment by LiadOvdat5** [comment]
> good job! do you need any help with the linting warnings?

Yes I would love that, 
I have warnings on the any in the interface 

image

image


what should I put there instead?

**Comment by supervxn** [comment]
@LiadOvdat5 - the frontend side looks clear and simple. 😺.

**Review by NoamGaash — APPROVED** [review]
Sorry for the delay, seems like you solved all linting issues + the code seems great!
We would love to have this merged. Thank you! :clap: :1st_place_medal:

**Review by NoamGaash — APPROVED** [review]
some comments - can be fixed after merge, or be opened as a new issue

File: src/pages/components/map-related/MapLayers/ComplaintModal.tsx [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+    position: 'absolute' as const,
+    top: '50%',
+    left: '50%',
+    transform: 'translate(-50%, -50%)',
+    width: 400,
+    bgcolor: 'background.paper',
+    border: '2px solid #000',
+    boxShadow: 24,
+    p: 4,
+    textAlign: i18n.language === 'he' ? 'left' : 'right',
```
Comment (NoamGaash) [inline src/pages/components/map-related/MapLayers/ComplaintModal.tsx:60]: something feels weird about this condition, are you sure it works as intended?

**Comment by supervxn** [comment]
Loving the speed here 🙌.
Let me know when it's live.

**Comment by NoamGaash** [comment]
@supervxn Once the pull request is merged (you'll probably get a notification about it) the deployment process will start automatically. It takes 10 minutes on average, but sometime (rarley) it gets stuck for about a day

**Comment by NoamGaash** [comment]
@supervxn 
From now on previews are also available on forks, so you can see the feature live.

**Comment by NoamGaash** [comment]
@all-contributors please add @LiadOvdat5 for code

**Comment by supervxn** [comment]
> @supervxn From now on previews are also available on forks, so you can see the feature live.

How can I view the fork?

**Comment by NoamGaash** [comment]
@supervxn have you seen this comment?
image
https://github.com/hasadna/open-bus-map-search/pull/791#issuecomment-2175075604

**Comment by supervxn** [comment]
> Preview: https://s3.amazonaws.com/noam-gaash.co.il/9559894563/open-bus/6/index.html Preview Storybook: https://s3.amazonaws.com/noam-gaash.co.il/9559894563/open-bus/6/storybook/index.html

I type complaint and nothing happens

**Comment by NoamGaash** [comment]
did you click on a bus icon inside the real-time map page before typing complaint?
did you use english keyboard?

**Comment by NoamGaash** [comment]
Also, now when this PR is merged, you can see it on production
https://open-bus-map-search.hasadna.org.il/map

**Comment by supervxn** [comment]
Got it! Thanks

**Comment by NoamGaash** [comment]
not yet

**Comment by supervxn** [comment]
> not yet

Phew.

**Pull request merged**